src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. Consider using a more descriptive name, for example `styleObject` or `cssStyles`.